packages/generators/generators/src/index.ts

```diff
@@ -1,28 +1,33 @@
 import { join } from 'node:path';
 import { Plop, run } from 'plop';
```

Comment:
isn't it ESM only now ?

Reply:
Yeah, not sure why this was working. I'm getting a unit test error that seems related, will figure it out and update